Makefile

```diff
@@ -1,24 +1,22 @@
 NXDK_DIR = $(CURDIR)/../nxdk
 NXDK_NET = y
 
-XBE_TITLE = triangle
+XBE_TITLE = nxdk-rdt
 GEN_XISO = $(XBE_TITLE).iso
 SRCS = $(wildcard $(CURDIR)/*.c)
 SHADER_OBJS = ps.inl vs.inl
 
 SRCS += $(CURDIR)/lib/protobuf-c/protobuf-c.c
 
-CFLAGS_EXTRA += -I$(CURDIR)
-CFLAGS_EXTRA += -I$(CURDIR)/lib
+CFLAGS += -I$(CURDIR)
+CFLAGS += -I$(CURDIR)/lib
 
 include $(NXDK_DIR)/Makefile
 
-.PHONY:clean_local
-clean_local:
+.PHONY:clean-local
+clean-local: clean
```

Comment:
In Makefiles you can add dependencies to an existing target defined elsewhere. So, in line 20 below that was removed, we added a dependency `clean_local` to the `clean` target, which would have removed the local stuff before doing a global clean. That way you can do `make clean` and it'll clean everything. If we do it the way you suggested here, a `make clean` wouldn't actually clean the local stuff, and a `make clean-local` would clean local and global. I prefer to keep it how it was before, unless there is a problem with it?